### PR TITLE
ART-7802 Monitor PR status; notify on merge/close; timeout after 1 week

### PR DESCRIPTION
**Pull request by locriandev**
Tested with:
```
# Open
watch https://github.com/openshift/release/pull/43595

# Merged
watch https://github.com/openshift/assisted-installer/pull/724

# Closed
watch https://github.com/openshift/assisted-installer/pull/704

# Non existent PR
watch https://github.com/openshift/assisted-installer/pull/7042344242

# Non existent repo
watch https://github.com/openshift/assisted-installerrrrrrrrr/pull/704

# ART repo
watch https://github.com/openshift-eng/art-bot/pull/152
watch https://github.com/openshift-eng/art-bot/pull/153

# Any repo
watch https://github.com/kubernetes/kubernetes/pull/120797
```

### CODE REVIEW

**Review by joepvd — COMMENTED** [review]
/lgtm

Let's wait a bit with deploying, there is a few watch threads running now :)

File: artbotlib/regex_mapping.py [hunk 1]
```diff
@@ -196,6 +197,12 @@ def map_command_to_regex(so, plain_text, user_id):
             "flag": re.I,
             "function": first_prow_job_succeeds,
             "user_id": True
+        },
+        {
+            "regex": r"^Watch \s*(https://)*(github.com/)*(openshift/)*(?P<repo>[a-zA-Z0-9-]+)(/pull/)(?P<pr_id>\d+)",
```
Comment (thegreyd) [inline artbotlib/regex_mapping.py:202]: Why not any github pr :nerd_face: ?

Comment (locriandev) [reply artbotlib/regex_mapping.py]: Sounds good, added in https://github.com/openshift-eng/art-bot/pull/153/commits/c070c7674a45b03bd74f1d2fb8096a76bea85574

**Pull request merged**